ui/js/components/loading.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+treeherder.component('loading',{
+    bindings: {
+        data: '<'
+    },
+    template: `
+        <div ng-if="$ctrl.data" class="overlay">
+            <div>
```

Comment:
(replying to myself: yes we do, because we need a div to have a relative positioning so we can position the spinner "absolutely" in its center)